pkgs/applications/misc/masterpdfeditor/default.nix

```diff
@@ -1,32 +1,32 @@
-{ stdenv, fetchurl, sane-backends, qtbase, qtsvg, autoPatchelfHook }:
+{ stdenv, fetchurl, sane-backends, qtbase, qtsvg, nss, autoPatchelfHook }:
 let
-  version = "4.3.89";
+  version = "5.1.00";
 in stdenv.mkDerivation {
   name = "masterpdfeditor-${version}";
   src = fetchurl {
     url = "https://code-industry.net/public/master-pdf-editor-${version}_qt5.amd64.tar.gz";
-    sha256 = "0k5bzlhqglskiiq86nmy18mnh5bf2w3mr9cq3pibrwn5pisxnxxc";
+    sha256 = "1s2zhx9xr1ny5s8hmlb99v3z1v26vmx87iixk8cdgndz046p9bg9";
   };
 
   nativeBuildInputs = [ autoPatchelfHook ];
 
-  buildInputs = [ sane-backends qtbase qtsvg ];
+  buildInputs = [ nss qtbase qtsvg sane-backends stdenv.cc.cc ];
```

Comment:
Why is `stdenv.cc.cc` needed? The package seems to build without it.

Reply:
with `nss` not yet inside `buildInputs`, `autoPatchelfHook` complained first about not being able to find `libstdc++.so.6` - that's why I added it.
Seems like this gets pulled in via `nss` too  - not sure if we still want to specify it explicitly…

Reply:
Fine then, let's keep it as is.